wormhole-connect/src/config/mainnet/tokens.ts

```diff
@@ -395,7 +395,7 @@ export const MAINNET_TOKENS: TokensConfig = {
   },
   ETHarbitrum: {
     key: 'ETHarbitrum',
-    symbol: 'ETH',
+    symbol: 'ETH (Arbitrum)',
```

Comment:
Symbols can't be more than 6 characters.  You'll need to add a displayName field